### PR TITLE
Generate certs

### PR DESCRIPTION
**Pull request by piotrmiskiewicz**

### CODE REVIEW

File: chart/templates/webhook.yml [hunk 1]
```diff
@@ -6,6 +33,9 @@ metadata:
     cert-manager.io/inject-ca-from: {{.Release.Namespace}}/sap-btp-operator-serving-cert
   {{- end}}
   name: sap-btp-operator-mutating-webhook-configuration
+  labels:
+    alt1: {{ $altName1 }}
+    alt2: {{ $altName2 }}
```
Comment (piotrmiskiewicz) [inline chart/templates/webhook.yml:38]: need to remove it

File: chart/templates/deployment.yml [hunk 2]
```diff
@@ -14,9 +14,8 @@ spec:
     metadata:
       annotations:
         {{- $configmap := (include (print $.Template.BasePath "/configmap.yml") .) -}}
-        {{- $secret := (include (print $.Template.BasePath "/secret.yml") .) -}}
-        {{- $secretTls := (include (print $.Template.BasePath "/secret-tls.yml") .) -}}
-        {{- $configSha := (print $configmap $secret $secretTls) | sha256sum }}
+          {{- $webhook := (include (print $.Template.BasePath "/webhook.yml") .) -}}
```
Comment (wozniakjan) [inline chart/templates/deployment.yml:17]: nit: is the indentation correct here?

**Review by wozniakjan — APPROVED** [review]
/lgtm
/hold

feel free to unhold if my hunch about the indent is not correct

**Pull request merged**